lib/notifier.js

```diff
@@ -53,6 +58,10 @@ class Notifier {
             notificationWindow.close()
         }, 4000)
 
+        const self = electron.remote && electron.remote.BrowserWindow.getCurrentWindow();
+        if (self)
+          self.on('close', notificationWindow.removeAllListeners.bind(notificationWindow));
```

Comment:
I'm not sure why we need this.

It feels similar to:

~~~ js
notificationWindow.on('close', () => {
  notificationWindow.removeAllListeners()
})
~~~

but even then, won't the events unbind themselves?